src/main/lrsql/system/openapi.clj

```diff
@@ -0,0 +1,12 @@
+(ns lrsql.system.openapi
+  (:require 
+   [com.yetanalytics.gen-openapi.generate.schema :as gs]
+   [com.yetanalytics.lrs.pedestal.openapi :as lrsoa]))
+
+(def oa-routes (atom nil))
+(def general-map {:openapi "3.0.0"
+                  :info {:title "LRSQL"
+                         :version "0.7.2"}
```

Comment:
is this supposed to be the release version? It's kinda rough when the release version needs to be manually updated on each push. There ARE solutions where we can introspect git repo on build for this, but they can be hairy.